syntax = "proto3";

message LocationMessage {
  int32 id = 1;
  int32 person_id = 2;
  string longitude = 3;
  string latitude = 4;
  string created_at = 5;
}

message LocationIdMessage {
  int32 id = 1;
}

message Empty {

}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
    rpc Get(LocationIdMessage) returns (LocationMessage);
}